includes/wikia/controllers/HeliosHelperController.class.php

```diff
@@ -154,28 +170,82 @@ public function sendTemporaryPasswordEmail() {
 
 		$user = \User::newFromName( $username );
 
-		if ( ! $user instanceof \User ) {
+		if ( !$user instanceof \User ) {
 			$this->response->setVal( 'message', 'unable to create a \User object from name' );
 			$this->response->setCode( \WikiaResponse::RESPONSE_CODE_BAD_REQUEST );
+
 			return;
 		}
 
-		if ( ! $user->getId() ) {
+		if ( !$user->getId() ) {
 			$this->response->setVal( 'message', 'no such user' );
 			$this->response->setCode( \WikiaResponse::RESPONSE_CODE_BAD_REQUEST );
+
 			return;
 		}
 
 		$resp = \F::app()->sendRequest( 'Email\Controller\ForgotPassword', 'handle', [
 			'targetUser' => $username,
-			'tempPass' => $tempPassword,
+			'tempPass'   => $tempPassword,
 		] );
 
 		$data = $resp->getData();
 		if ( !empty( $data['result'] ) && $data['result'] == 'ok' ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "result" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)